app/Dockerfile

```diff
@@ -52,6 +52,7 @@ RUN chmod +x /usr/local/bin/dumb-init
 ENTRYPOINT ["dumb-init", "--", "/usr/bin/env", "bash", "-c"]
 
 COPY app/ ${APP_HOME}/
+COPY mocks_schema/ ${HOME}/mocks_schema
```

Comment:
@VB-at-Bis I didn't pay attention... This is in the app Dockerfile, it should be in the db Dockerfile